functional-tests/affected-ftest/pom.xml

```diff
@@ -42,6 +42,9 @@
           <properties>
             <orderStrategy>affected</orderStrategy>
           </properties>
+          <includes>
+            <include>**/src/test/java/**/Test*.java</include>
```

Comment:
Let's make it more generic, e.g. `*Test*.java`, but then we should avoid having `TestUtils` or something similar ;)